unirec/unirec.c

```diff
@@ -368,21 +375,29 @@ char *ur_ifc_data_fmt_to_field_names(const char *ifc_data_fmt)
       //copy name
       source_cpy = p;
       name_len = 0;
-      while (*p != 0 && *p != ',') {
+      while (*p != 0 && *p != ',' && !isspace(*p)) {
          name_len++;
          p++;
       }
       assert(name_len + act_len + 1 <= str_len);
       memcpy(out_str + act_len, source_cpy, name_len);
       act_len += name_len;
-      if (*p == 0) {
-         break;
+      /* skip white spaces */
+      while (*p != 0 && isspace(*p)) {
+         p++;
       }
-      out_str[act_len] = ',';
-      act_len++;
       if (*p == ',') {
          p++;
       }
+      else if (*p == 0) {
```

Comment:
This is against [coding style](https://github.com/CESNET/Nemea-Framework/blob/master/libtrap/CodingStyle.md#statements)